src/connector/src/parser/util.rs

```diff
@@ -25,6 +25,20 @@ use crate::connector_common::AwsAuthProps;
 use crate::error::ConnectorResult;
 use crate::source::SourceMeta;
 
+#[macro_export]
```

Comment:
We may not export this macro. Can add a line of `pub(crate) use log_error` to make it available in other modules in this crate.

Reply:
updated